shared/test/fixtures/vcr/files/upload_files.yml

```diff
@@ -4,36 +4,47 @@ http_interactions:
     method: get
```

Comment:
Mostly out of curiosity- how is this file generated?

Reply:
background: we're using a tool called VCR that records all of the network requests made in a given test file. once the request is in this file, we can just "replay" the request in subsequent test runs (hence the VCR name). there's a little [README](https://github.com/code-dot-org/code-dot-org/blob/staging/shared/test/README.md) about it here.

more specifically: when i added my test, VCR yelled at me about there being a new/different network request, so i deleted this YML file and re-ran the tests to regenerate the file with my added requests